frontend/app/src/people/widgetViews/organization/HistoryModal.tsx

```diff
@@ -53,33 +294,93 @@ const HistoryModal = (props: PaymentHistoryModalProps) => {
         borderRadius: '50%'
       }}
     >
-      <OrgWrap style={{ width: '300px' }}>
-        <ModalTitle>Payment history</ModalTitle>
-        <table>
-          <thead>
-            <tr>
-              <th>Sender</th>
-              <th>Recipient</th>
-              <th>Amount</th>
-              <th>Date</th>
-              <th />
-            </tr>
-          </thead>
-          <tbody>
-            {paymentsHistory.map((pay: PaymentHistory, i: number) => (
-              <tr key={i}>
-                <td className="ellipsis">{pay.sender_name}</td>
-                <td className="ellipsis">{pay.receiver_name}</td>
-                <td>{pay.amount} sats</td>
-                <td>{moment(pay.created).format('DD/MM/YY')}</td>
-                <td>
-                  <ViewBounty onClick={() => viewBounty(pay.bounty_id)}>View bounty</ViewBounty>
-                </td>
-              </tr>
-            ))}
-          </tbody>
-        </table>
-      </OrgWrap>
+      <HistoryWrapper>
+        <ModalHeaderWrapper>
+          <ModalTitle>Payment History</ModalTitle>
+          <PaymentFilterWrapper>
+            <PaymentType>
+              <input
+                id="payment"
+                type={'checkbox'}
+                checked={filter.payment}
+                onChange={() => handleFilter('payment')}
+              />{' '}
```

Comment:
not sure why `{' '}` is here

Reply:
Hm my editor always adds this, I will remove it manually

Reply:
Hm my editor always adds this, I will remove it manually